R/teal_slices-store.R

```diff
@@ -1,23 +1,28 @@
-#' Store `teal_slices` object to a file
+#' Store and restore `teal_slices` object
 #'
-#' This function takes a `teal_slices` object and saves it to a file in `JSON` format.
+#' Functions that serialize a `teal_slices` object to and from a file in
+#' `JSON` format.
 #' The `teal_slices` object contains information about filter states and can be used to
-#' create, modify, and delete filter states. The saved file can be later loaded using
-#' the `slices_restore` function.
+#' create, modify, and delete filter states.
 #'
-#' `Date` class is stored in `"ISO8601"` format (`YYYY-MM-DD`). `POSIX*t` classes are converted to a
-#' character by using `format.POSIX*t(usetz = TRUE, tz = "UTC")` (`YYYY-MM-DD {N}{N}:{N}{N}:{N}{N} UTC`, where
-#' `{N} = [0-9]` is a number and `UTC` is `Coordinated Universal Time` timezone short-code).
-#' This format is assumed during `slices_restore`. All `POSIX*t` objects in `selected` or `choices` fields of
-#' `teal_slice` objects are always printed in `UTC` timezone as well.
+#' Date formats are converted in the following way:
 #'
-#' @param tss (`teal_slices`) object to be stored.
-#' @param file (`character(1)`) The file path where `teal_slices` object will be saved.
-#'  The file extension should be `".json"`.
+#' - `Date` class is stored in `"ISO8601"` format (`YYYY-MM-DD`).
+#' - `POSIX*t` classes are converted to a character by using
+#' `format.POSIX*t(usetz = TRUE, tz = "UTC")` (`YYYY-MM-DD {N}{N}:{N}{N}:{N}{N} UTC`,
+#' where `{N} = [0-9]` is a number and `UTC` is `Coordinated Universal Time`
+#' timezone short-code).
+#'
+#' This format is assumed during `slices_restore`. All `POSIX*t` objects in
+#' `selected` or `choices` fields of `teal_slice` objects are always printed in
+#' `UTC` timezone as well.
 #'
-#' @return `NULL`, invisibly.
+#' @param tss (`teal_slices`) object to be stored.
+#' @param file (`character(1)`) The file path where `teal_slices` object will be
+#' saved and restored. The file extension should be `".json"`.
 #'
-#' @examples
+#' @return `slices_store` returns `NULL`, invisibly.
```

Comment:
Return values can be collated under one tag.

Reply:
I prefer to keep the function-specific tags separate as much as possible. Do you think we should collate both of them in `slices_store`?

Reply:
Since this is going in one file and one page, it makes sense to me that there is only one `@return` tag. Every other tag is collated. But it's a small file so it's not a big deal in this case.